wormhole-connect/src/views/Bridge/Inputs/AmountInput.tsx

```diff
@@ -53,16 +62,19 @@ function AmountInput(props: Props) {
       cursor="text"
     >
       {token ? (
-        <InputTransparent
-          inputRef={amountEl}
-          placeholder="0.00"
-          type="number"
-          min={0}
-          step={0.1}
-          onChange={handleAmountChange}
-          disabled={isTransactionInProgress || props.disabled}
-          value={props.value}
-        />
+        <>
+          <InputTransparent
+            inputRef={amountEl}
+            placeholder="0.00"
+            type="number"
+            min={0}
+            step={0.1}
+            onChange={handleAmountChange}
+            disabled={isTransactionInProgress || props.disabled}
+            value={props.value}
+          />
+          {price && <Price>{`$${price}`}</Price>}
```

Comment:
what about the price component accepting a value and rendering it if the value is different than null?

Reply:
if the value is null it doesn't show anything, otherwise it shows the value with the expected style